app/main.js

```diff
@@ -1,4 +1,81 @@
 var app = require('electron').app; // Module to control application life.
+var childProcess = require("child_process");
```

Comment:
Probably worthwhile to move squirrel events and shortcut creation into another file..